Dockerfile

```diff
@@ -1,14 +1,20 @@
 FROM nginx:1.10
 MAINTAINER Wyatt Pearsall<Wyatt.Pearsall@cfpb.gov>
 
-RUN mkdir -p /usr/src/app
+RUN apt-get update && \
```

Comment:
If we want only one Dockerfile, then yeah. We can bring it up in discussion today.